cmd/up.go

```diff
@@ -350,16 +351,22 @@ func (cmd *UpCmd) devPodUpMachine(
 		return nil, err
 	}
 
+	// ssh tunnel
+	sshCmd := fmt.Sprintf("'%s' helper ssh-server --stdio", client.AgentPath())
+	if log.GetLevel() == logrus.DebugLevel {
+		sshCmd += " --debug"
+	}
+
 	// create container etc.
 	log.Infof("Creating devcontainer...")
```

Comment:
can you create the agent command where we actually execute it or is there a reason to do it here?

Reply:
Done.